src/color.rs

```diff
@@ -310,7 +310,7 @@ fn f64_bound(min: f64, val: f64, max: f64) -> f64 {
     debug_assert!(min.is_finite());
     debug_assert!(val.is_finite());
     debug_assert!(max.is_finite());
```

Comment:
```suggestion
    debug_assert!(val.is_finite());
```
I don't think the `f64::INFINITY` case is likely to come up for `min` or `max`.

(Fwiw, I think the last assertion is a bit dodgy; infinity is fine, it's just `NaN` that we don't want to propagate)

Reply:
I agree that one is a bit dodgy, infinity would get clamped to the almost-surely-finite `min` or `max` anyway.

Reply:
I'll just merge it as-is and not spend too much thought on it, as it's likely to go away anyway. This keeps the same behavior, which may be useful for the patch release.